lib/v8-log-to-ticks.js

```diff
@@ -122,7 +122,10 @@ function v8LogToTicks (isolateLogPath, node) {
     const tickStream = parse('ticks.*', (tick) => {
       const addr = tick.s.filter((n, i) => i % 2 === 0)
       var stack = addr.map((n) => codes[n]).filter(Boolean)
-      ticks.push(stack.reverse())
+      const delayMs = options.collectDelay * 1000
+      if (tick.tm > (tick.tm+delayMs)) {
```

Comment:
this will always evaluate to `false` :sweat_smile: The callback to `parse()` is called once for each tick. we need to remember the value for the first tick and compare to that in all subsequent ticks.

Reply:
Yeah I realized that pretty quick. :laughing: I've another commit up for it.